### ISSUE TITLE
Fix false positives for attached additional props

### ISSUE THREAD

**Issue by jvega** [stylelint/stylelint#3710]
> Clearly describe the bug

Attaching additional props cause false negatives
https://www.styled-components.com/docs/basics#attaching-additional-props

I think is related with #3690

> What CSS is needed to reproduce the bug?

```js
import styled from 'styled-components'

const Component = styled.div.attrs({
  className: 'someClassName'
})`
  color: red;
`

export default Component

```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": [
    "stylelint-config-standard"
  ]
}
```

> Which version of stylelint are you using?

`9.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

CSS-in-JS

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

"The following warnings were flagged:"

```shell
 4:2  ✖  Unexpected unknown property "class-name"   property-no-unknown
```

**Comment by jvega** [comment]
@jeddy3 yes I think should be ignored because all the code there could be JS code

**Comment by jeddy3** [comment]
@jvega Thanks for the clarification. In that case, I'll label this as a bug. It'll likely need to be fixed upstream in the [`postcss-styled`](https://github.com/gucong3000/postcss-styled) package. If you'd like to see this resolved quickly, please consider helping out and contributing a fix to that package.

**Comment by gucong3000** [comment]
Fixed in #3869
